### PR TITLE
Displayed the attribute value as comma-separated(#1xc2cfg)

### PR DESCRIPTION
**Pull request by disha1202**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

Screenshot from 2021-12-29 16-41-57

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]
Add changelog file

File: src/main.ts [hunk 1]
```diff
@@ -75,7 +75,8 @@ app.config.globalProperties.$filters = {
       featureHierarchy.filter((featureItem: any) => featureItem.startsWith(featureKey)).forEach((feature: any) => {
         const featureSplit = feature ? feature.split('/') : [];
         const featureValue = featureSplit[2] ? featureSplit[2] : '';
-        featuresValue +=  " " + featureValue;
+        featuresValue += ", " + featureValue;
+        featuresValue = featuresValue.slice(1);
```
Comment (ymaheshwari1) [inline src/main.ts:79]: I think we should find a much better approach to add a comma between values.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/main.ts (hunk 1 above)
Comment (disha1202) [reply src/main.ts]: Yes Sir. Updated code with a better approach.

**Comment by disha1202** [comment]
> Add changelog file

Added a changelog file.

**Review by ymaheshwari1 — APPROVED** [review]

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**